pyscal/factory.py

```diff
@@ -553,7 +553,11 @@ def create_scal_recommendation(
             raise ValueError('"base" case not supplied')
         if "high" not in params:
             raise ValueError('"high" case not supplied')
-        if not all([isinstance(x, dict) for x in params.values()]):
+        if not (
+            isinstance(params["low"], dict)
+            and isinstance(params["base"], dict)
+            and isinstance(params["high"], dict)
+        ):
```

Comment:
Did `pylint` really prefer this over the more compact old one, or is it enough to remove the `[ ]` as you have done later in similar cases?

Reply:
pylint didn't like the list generator, that could have been fixed by removing square brackets, but then I found the explicit way of writing it more readable.